site/pages/admin/social-media-accounts.page.tsx

```diff
@@ -83,7 +86,11 @@ const SocialMediaAccounts = ({
                 {socialAccountToDelete ? (
                     <DeleteConfirmationPopup
                         entityName={`the ${
```

Comment:
Think we could simplify this to:
``entityName={`the ${socialAccountToDelete.accountType.toLowerCase()} connection`}``

Reply:
done in https://github.com/Department-for-Transport-Disruptions/create-disruptions-data/pull/418